src/autocomplete.css

```diff
@@ -17,28 +17,40 @@
 .autocomplete__input {
   background-color: transparent;
   position: relative;
+  padding: 5px 35px 5px 5px; /* Space for arrow */
 }
 
 .autocomplete__hint {
   color: #505a5f;
   position: absolute;
 }
 
-.autocomplete__input--default {
-  padding: 5px;
-}
-
 .autocomplete__input--focused {
   outline: 3px solid #fd0;
   outline-offset: 0;
   box-shadow: inset 0 0 0 2px;
 }
 
 .autocomplete__input--show-all-values {
-  padding: 5px 35px 5px 5px; /* Space for arrow. Other padding should match .autocomplete__input--default. */
   cursor: pointer;
 }
 
+.autocomplete__input--bottomless {
+  border-bottom: 0;
+}
+
+.autocomplete__input--bottomless.autocomplete__input--focused {
+  box-shadow: inset 2px 2px 0 0, inset -2px 0 0 0;
+}
+
+.autocomplete__dropdown-arrow-down-wrapper {
+  display: none;
+}
+
+.autocomplete__dropdown-arrow-down-wrapper--visible {
+  display: block;
+}
```

Comment:
**question** Given the `dropdownArrowFactory` accepts a `className`, what's the reason for adding an extra wrapper element with its own class rather than using a `className` with two class names separated by a space?

Reply:
See https://github.com/alphagov/accessible-autocomplete/pull/753/files#r1791991077 as well. It'd be neat if we could only use the existing `className` prop to control the arrow's visibility.

Another option could be to set a class on a wrapping element and target the `dropdownArrowClassName` we set to control visibility.